### PR TITLE
More debug text fixes

### PR DESCRIPTION
**Pull request by JosJuice**
Also a few small non-debug text changes.

---
This change is [Reviewable](https://reviewable.io/reviews/dolphin-emu/dolphin/4412)

### CODE REVIEW

File: Source/Core/DolphinWX/Debugger/WatchView.cpp [hunk 1]
```diff
@@ -256,6 +257,7 @@ void CWatchView::OnMouseDownR(wxGridEvent& event)
     TryParse("0x" + WxStrToStr(strNewVal), &m_selectedAddress);
   }
 
+  // i18n: Not the kind of watch that shows time
```
Comment (BhaaLseN) [inline Source/Core/DolphinWX/Debugger/WatchView.cpp]: This one looks like its in the wrong place?

File: Source/Core/DolphinWX/Debugger/MemoryView.cpp [hunk 2]
```diff
@@ -250,6 +250,7 @@ void CMemoryView::OnMouseDownR(wxMouseEvent& event)
   menu.Append(IDM_COPYADDRESS, _("Copy &address"));
   menu.Append(IDM_COPYHEX, _("Copy &hex"));
 #endif
+  // i18n: Not the kind of watch that shows time
```
Comment (degasus) [inline Source/Core/DolphinWX/Debugger/MemoryView.cpp]: same, at least some kind of "memory watch"

File: Source/Core/DolphinWX/Debugger/WatchView.cpp (hunk 1 above)
Comment (degasus) [inline Source/Core/DolphinWX/Debugger/WatchView.cpp]: for "wxMenu menu;"?

**Comment by degasus** [comment]
else LGTM

File: Source/Core/DolphinWX/Debugger/DSPDebugWindow.cpp [hunk 3]
```diff
@@ -46,8 +46,11 @@ DSPDebuggerLLE::DSPDebuggerLLE(wxWindow* parent, wxWindowID id)
       new DolphinAuiToolBar(this, ID_TOOLBAR, wxDefaultPosition, wxDefaultSize, wxAUI_TB_HORZ_TEXT);
   m_Toolbar->AddTool(ID_RUNTOOL, _("Pause"),
                      wxArtProvider::GetBitmap(wxART_TICK_MARK, wxART_OTHER, m_toolbar_item_size));
+  // i18n: Here, "Step" is a verb. This function is used for
+  // going through code step by step.
   m_Toolbar->AddTool(ID_STEPTOOL, _("Step"),
                      wxArtProvider::GetBitmap(wxART_GO_DOWN, wxART_OTHER, m_toolbar_item_size));
+  // i18n: Here, PC is an acronym for program counter, not personal computer.
```
Comment (degasus) [inline Source/Core/DolphinWX/Debugger/DSPDebugWindow.cpp]: Is this comment fine here?

**Pull request merged**